File: project/gateway/Models/driver_car.py
from pydantic import BaseModel


class DriverCar(BaseModel):
    id: int
    driver_id: int
    car_id: int
    start_date: str
    end_date: str
    place_departure: str
    place_destination: str
    distance: float
    duration: int
    fuel_consumption: float
    violations_per_trip: int
    average_speed: int

File: project/ui/models/driverCar.ts
export type DriverCar = {
    id: number
    driver_id: number
    car_id: number
    start_date: string
    end_date: string
    place_departure: string
    place_destination: string
    distance: number
    duration: number
    fuel_consumption: number
    violations_per_trip: number
    average_speed: number
}